navigation/screens/facility-reviews.tsx

```diff
@@ -1,18 +1,58 @@
 import React from "react";
-import { Button, Image, ScrollView, StyleSheet, Text, View } from "react-native";
+import {
+  Button,
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from "react-native";
+import {
+  likeReview,
+  dislikeReview,
+  replyReview,
+} from "@/controllers/reviewController";
+import { useState } from "react";
 
 export default function FacilityReviewsScreen() {
+  /**like a review */
+  const [reviewID, setReviewID] = useState("");
+  const [username, setUsername] = useState("");
```

Comment:
this is also not state. it will probably come from some sort of authentication context object that is passed into this component from it's parent